packages/ag-charts-community/src/scale/scale.ts

```diff
@@ -1,9 +1,5 @@
-interface ScaleClampParams {
-    /**
-     * If `true` the values outside of the domain will become `NaN`.
-     * If `false` such values will be clamped to the domain edges.
-     */
-    strict: boolean;
+export interface ScaleClampParams {
+    mode: 'clamped' | 'raw';
```

Comment:
nit: for future proofing we may wanna make the interface more generic, e.g. `ScaleConvertParams` & `clipMode`

Reply:
They are distinct concepts in my mind so wouldn't want to conflate them, but maybe I'm mis-understanding your suggestion?

Reply:
I'll try to clarify my suggestion.
Today the scale covert method signature is: `convert(value: D, params?: ScaleClampParams): R;`.
We explicitly declare the params of the covert method as being clamp params, but in a broader sense these are the convert method params which currently, for the time being, contain only the clamp mode option, but that may change in the future and hold other params which relate to the convert method, not necessarily clamp related. 
that's why I'm suggesting merely changing the names to make them more generic in advance. I also believe this reflect the intention better, so if tomorrow someone need another option they'll know in goes in those params and there's no need for yer another argument for it.
So instead of `ScaleClampParams` we could call it `ScaleConvertParams`, because these are params for the scale convert method, and instead of simply `mode` we could call it `clampMode` so the context will remain clear, that way we could still use the same params object to pass more options even if they're unrelated to clamping.

Reply:
Ok - I've renamed.